src/Schema/TypeRegistry.php

```diff
@@ -348,20 +348,20 @@ protected function resolveScalarType(ScalarTypeDefinitionNode $scalarDefinition)
 
         $namespacesToTry = (array) config('lighthouse.namespaces.scalars');
 
-        $className = Utils::namespaceClassname(
+        $classNameFQN = Utils::namespaceClassname(
```

Comment:
"class name fully qualified name" seems like a clunky term. Let's stick with the existing terminology.

```suggestion
        $namespacedClassName = Utils::namespaceClassname(
```